src/django_otp/tests.py

```diff
@@ -164,6 +165,45 @@ def test_verify_is_allowed(self):
             self.assertEqual(data3, None)
 
 
+class GenerationThrottlingTestMixin:
```

Comment:
It looks like this is based on `ThrottlingTestMixin`, which makes sense. However, it's asserting on `len(mail.outbox)`, which is email-specific. This will presumably need at least one abstract method to actually be generic.

Reply:
I agree. I have added generic tests and moved email specific tests to the email test file ([29ebe3e](https://github.com/django-otp/django-otp/pull/122/commits/29ebe3e64a5daca52a4272edb0cc4ef02df8526c)).